src/containerapp/azext_containerapp/_clients.py

```diff
@@ -25,7 +25,7 @@
 
 logger = get_logger(__name__)
 
-PREVIEW_API_VERSION = "2023-05-02-preview"
+PREVIEW_API_VERSION = "2023-08-01-preview"
```

Comment:
@Greedygre, @howang-ms > I assume this API version will be updated for everyone soon. I changed it here to access the new APIs, but I'd be curious to know if you have any visibility on when this will be updated?

Reply:
Hi @daniv-msft 

For the api-version update in azure-cli-extension, generally we need to wait for api-version public, PR for 2023-08-01-preview is: https://github.com/Azure/azure-rest-api-specs/pull/26352 merged.
After the api-version public, we will upgrade the model class to 2023-08-01-preview if need.

Reply:
Thank you @Greedygre! OK so once https://github.com/Azure/azure-rest-api-specs/pull/26352 is merged, then we should be good to update to 2023-08-01-preview in the CLI. Depending on when the API specs PR gets merged, I might have to force the use of 2023-08-01-preview for the BuilderClient/BuildClient API calls only as we need these changes available by end of this week. I'll keep you and @howang-ms posted if it's the case.

Reply:
@daniv-msft we should wait for 2023-08-01-preview api version manifest rollout to worldwide. The manifest rollout needs to take at least 5 days. So it is hard to support it by end of this week.

Reply:
Thanks @Juliehzl. @Greedygre and I were talking about this separately.
In this case what I might do is just using the new API version in the BuildClient/BuilderClient I'm introducing, and asking the partner team who needs to test these changes to register for PrereleaseApiVersionAllowed.
That way, it will unblock the partners without making this API version being rushed.